api-authorization/server/lib/authorization-middleware.js

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable no-unused-vars -- Remove me */
+import jwt from 'jsonwebtoken';
+import { ClientError } from './client-error.js';
+
+export function authMiddleware(req, res, next) {
+  /* your code here */
+  const auth = req.get('Authorization');
+  const token = auth.split('Bearer ')[1];
```

Comment:
This has a bug. What happens if there is no Authorization header?

Reply:
Unfortunately, you are checking `auth` for undefined after you've already dereferenced it!